packages/@aws-cdk/aws-redshift/lib/cluster.ts

```diff
@@ -0,0 +1,413 @@
+import * as ec2 from '@aws-cdk/aws-ec2';
+import { IRole } from '@aws-cdk/aws-iam';
+import * as kms from '@aws-cdk/aws-kms';
+import * as secretsmanager from '@aws-cdk/aws-secretsmanager';
+import { Construct, Duration, RemovalPolicy, Resource, Token } from '@aws-cdk/core';
+import { ClusterAttributes, ICluster } from './cluster-ref';
+import { DatabaseSecret } from './database-secret';
+import { Endpoint } from './endpoint';
+import { IParameterGroup } from './parameter-group';
+import { Login, RotationMultiUserOptions } from './props';
+import { CfnCluster, CfnClusterSubnetGroup } from './redshift.generated';
+
+/**
+ * What node type to use
```

Comment:
What be great to mention here what property uses this type ("used by `{@link ClusterProps.nodeType}`, and what is the default.

Reply:
What is a best practice, when `@link`ing things?
Is it a good idea to `@link` a lot? Like here
https://github.com/aws/aws-cdk/blob/0f0e2055cab08551bc6e5dfb8a1b6219368263c5/packages/%40aws-cdk/aws-codebuild/lib/artifacts.ts#L16-L20
or here
https://github.com/aws/aws-cdk/blob/6103180c5427bb887fe1734330b109579874649d/packages/%40aws-cdk/aws-codebuild/lib/project.ts#L1030-L1036

Reply:
Yeah, generally, links are a good idea, because they make the docs more helpful ([example](https://docs.aws.amazon.com/cdk/api/latest/typescript/api/aws-codebuild/iartifacts.html#aws_codebuild_IArtifacts)).

Note that links in `@default` do not actually work, so don't bother with them.